packages/p2p/src/components/my-profile/my-profile-form/my-profile-form.tsx

```diff
@@ -29,37 +29,37 @@ const MyProfileForm = () => {
                             {({ field }) => (
                                 <Input
                                     {...field}
-                                    type='textarea'
+                                    className='my-profile-form__textarea'
+                                    error={errors.contact_info}
+                                    has_character_counter
+                                    initial_character_count={general_store.contact_info.length}
                                     label={
                                         <Text color='less-prominent' size='xs'>
                                             <Localize i18n_default_text='Contact details' />
                                         </Text>
                                     }
-                                    error={errors.contact_info}
-                                    className='my-profile-form__textarea'
-                                    has_character_counter
-                                    initial_character_count={general_store.contact_info.length}
                                     max_characters={300}
+                                    type='textarea'
                                 />
                             )}
                         </Field>
                         <Field name='default_advert_description'>
                             {({ field }) => (
                                 <Input
                                     {...field}
-                                    type='textarea'
+                                    className='my-profile-form__textarea'
+                                    error={errors.default_advert_description}
+                                    has_character_counter
+                                    hint={localize('This information will be visible to everyone.')}
+                                    initial_character_count={general_store?.default_advert_description?.length}
```

Comment:
shouldn't this be `general_store?.default_advert_description?.length ?? 0`

Reply:
actually dont need to do that since in Input, even if initial_character_count returns null or undefined, counter is already set to 0 by default

![Screenshot 2023-06-08 at 3 56 26 PM](https://github.com/ameerul-deriv/deriv-app/assets/103412909/7d5b8fe4-4267-4e32-b6e6-1a9cc9989963)

Reply:
ah.. okay.. i didn't notice that.